examples/quickstart.py

```diff
@@ -1,78 +1,64 @@
 from imitation.data import rollout
-from imitation.rewards import reward_nets
-from imitation.util import logger, util
+from imitation.data.wrappers import RolloutInfoWrapper
+
+env = gym.make("CartPole-v1")
+
+
+def train_expert():
+    logging.info("Training a expert.")
+    expert = PPO(
+        policy=MlpPolicy,
+        env=env,
+        seed=0,
+        batch_size=64,
+        ent_coef=0.0,
+        learning_rate=0.0003,
+        n_epochs=10,
+        n_steps=64,
+    )
+    expert.learn(100000)
+    return expert
 
-# Load pickled test demonstrations.
-with open("tests/testdata/expert_models/cartpole_0/rollouts/final.pkl", "rb") as f:
-    # This is a list of `imitation.data.types.Trajectory`, where
-    # every instance contains observations and actions for a single expert
-    # demonstration.
-    trajectories = pickle.load(f)
 
-# Convert List[types.Trajectory] to an instance of `imitation.data.types.Transitions`.
-# This is a more general dataclass containing unordered
-# (observation, actions, next_observation) transitions.
-transitions = rollout.flatten_trajectories(trajectories)
+def sample_expert_transitions():
+    expert = train_expert()
 
-venv = util.make_vec_env("seals/CartPole-v0", n_envs=2)
+    logging.info("Sampling expert transitions.")
+    rollouts = rollout.rollout(
+        expert,
+        DummyVecEnv([lambda: RolloutInfoWrapper(env)]),
+        rollout.make_sample_until(min_timesteps=None, min_episodes=50),
+    )
+    return rollout.flatten_trajectories(rollouts)
 
-tempdir = tempfile.TemporaryDirectory(prefix="quickstart")
-tempdir_path = pathlib.Path(tempdir.name)
-print(f"All Tensorboards and logging are being written inside {tempdir_path}/.")
 
-# Train BC on expert data.
-# BC also accepts as `demonstrations` any PyTorch-style DataLoader that iterates over
-# dictionaries containing observations and actions.
-bc_logger = logger.configure(tempdir_path / "BC/")
+transitions = sample_expert_transitions()
 bc_trainer = bc.BC(
-    observation_space=venv.observation_space,
-    action_space=venv.action_space,
+    observation_space=env.observation_space,
+    action_space=env.action_space,
     demonstrations=transitions,
-    custom_logger=bc_logger,
 )
-bc_trainer.train(n_epochs=1)
 
-# Train GAIL on expert data.
-# GAIL, and AIRL also accept as `demonstrations` any Pytorch-style DataLoader that
-# iterates over dictionaries containing observations, actions, and next_observations.
-gail_logger = logger.configure(tempdir_path / "GAIL/")
-gail_reward_net = reward_nets.BasicRewardNet(
-    observation_space=venv.observation_space,
-    action_space=venv.action_space,
-)
-gail_trainer = gail.GAIL(
-    venv=venv,
-    demonstrations=transitions,
-    demo_batch_size=32,
-    gen_algo=sb3.PPO("MlpPolicy", venv, verbose=1, n_steps=1024),
-    reward_net=gail_reward_net,
-    custom_logger=gail_logger,
-)
-gail_trainer.train(total_timesteps=2048)
+reward, _ = evaluate_policy(bc_trainer.policy, env, 3, render=True)
```

Comment:
Consider `n_eval_episodes=3` to make it explicit (`3` seems quite magical otherwise); same for next usage on line 63

Reply:
Good point. I fixed it.